src/containers/ResetPasswordForm/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { resetPasswordAction } from './state/actions';
+import ROUTES from '../../utils/routes';
+import { getQueryParams } from '../../utils/helpers';
+
+class ResetPasswordForm extends Component {
+  state = {
+    email: null,
+    password: null,
+    confirmPassword: null,
+  };
+
+  handleUpdate = (ev) => {
```

Comment:
Make the parameter `ev` more descriptive

Reply:
Done